src/config/env.zod.ts

```diff
@@ -0,0 +1,117 @@
+import { z, type ZodTypeAny } from "zod";
+
+//
+export const isTrue = (v: string) => v === "True";
+export const zodTrueFalseBoolean = z
+  .enum(["True", "False"])
+  .default("False")
+  .transform(isTrue)
```

Comment:
isTrue retourne déjà un booléen non ?

Reply:
I remove the `.pipe(z.coerce.boolean());` ?